upgrade/upd-2.4.0-to-2.4.1/index.php

```
@@ -70,19 +70,48 @@ public function apply_license()
     public function xoops_upgradeLicenseKey($public_key, $licensefile, $license_file_dist = 'license.dist.php')
     {
-        chmod($licensefile, 0777);
-        $fver        = fopen($licensefile, 'w');
-        $fver_buf    = file($license_file_dist);
+        $fver_buf = file($license_file_dist);
+        if (false === $fver_buf) {
+            return false;
+        }
         $license_key = $public_key . substr(XOOPS_LICENSE_KEY, 13, strlen(XOOPS_LICENSE_KEY) - 13);
+        $content = '';
         foreach ($fver_buf as $line => $value) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$line'. (undefined)

(UnusedLocalVariable)